inst/rmarkdown/templates/hugo/resources/default.md

```diff
@@ -7,6 +7,9 @@ title: $if(title)$$title$$endif$
 categories: Data Science
 tags: 
   - R
+  - $if(tag1)$$tag1$$endif$
+  - $if(tag2)$$tag2$$endif$
+image: $if(image)$$image$$endif$
```

Comment:
Even better:
```
$if(image)$image:$endif$ $if(image)$$image$$endif$
```
then do the same for tag1 and tag2, so they only show up if the user adds them.